### DISCUSSION TITLE
publish the code that pin discussions to be used in website

### DISCUSSION THREAD

**Discussion by prudhomm** [community/community#63899] [Discussions]
### Select Topic Area

Product Feedback

### Body

It would be nice to be able to embed discussions in a website eg to have a blog or announcements or latest news added to the website automatically updated.
It would be nice to have a timeline of discussions per category or pinned and used for example the way they are presented as pinnned on discussions and embed them on the website.
I would think that this would be very helpful to make discussions even more used by developers on GitHub

**Comment by Daeraxa** [comment]
Like this? https://github.com/orgs/community/discussions/56489
